### PR TITLE
Fix for unequal nulls bitmap size YQL-17564

### PR DESCRIPTION
**Pull request by aakulaga-ydb**
- Fix for unequal nulls bitmap size YQL-17564

### CODE REVIEW

File: ydb/library/yql/minikql/comp_nodes/mkql_grace_join_imp.cpp [hunk 1]
```diff
@@ -417,14 +416,15 @@ void TTable::Join( TTable & t1, TTable & t2, EJoinKind joinKind, bool hasMoreLef
             auto slotIt = joinSlots.begin() + slotNum * slotSize;
             while (*slotIt != 0 && slotIt != joinSlots.end())
             {
+
                 bool matchFound = false;
-                if (keysValSize <= slotSize && !JoinTable1->NumberOfKeyIColumns ) {
+                if (((keysValSize - nullsSize1) <= (slotSize - nullsSize2)) && !JoinTable1->NumberOfKeyIColumns ) {
```
Comment (resetius) [inline ydb/library/yql/minikql/comp_nodes/mkql_grace_join_imp.cpp:421]: better
```
keysValSize + nullsSize2 <= slotSize + nullsSize1
```

File: ydb/library/yql/minikql/comp_nodes/mkql_grace_join_imp.cpp [hunk 2]
```diff
@@ -417,14 +416,15 @@ void TTable::Join( TTable & t1, TTable & t2, EJoinKind joinKind, bool hasMoreLef
...
                     if (std::equal(it1 + keyIntOffset1, it1 + keysValSize, slotIt + keyIntOffset2)) {
                         matchFound = true;
                     }
                 }
 
-                if (keysValSize > slotSize && !JoinTable1->NumberOfKeyIColumns ) {
+                if (((keysValSize - nullsSize1) > (slotSize - nullsSize2)) && !JoinTable1->NumberOfKeyIColumns ) {
```
Comment (resetius) [inline ydb/library/yql/minikql/comp_nodes/mkql_grace_join_imp.cpp:427]: better
```
keysValSize + nullsSize2 > slotSize + nullsSize1
```

File: ydb/library/yql/minikql/comp_nodes/mkql_grace_join_imp.cpp (hunk 1 above)
Comment (aakulaga-ydb) [reply ydb/library/yql/minikql/comp_nodes/mkql_grace_join_imp.cpp:421]: keysValSize is related to probe table 1 (nullsSize1).  slotSize is related to hash table (nullsSize2).  So it is more natural.

File: ydb/library/yql/minikql/comp_nodes/mkql_grace_join_imp.cpp (hunk 2 above)
Comment (aakulaga-ydb) [reply ydb/library/yql/minikql/comp_nodes/mkql_grace_join_imp.cpp:427]: keysValSize is related to probe table 1 (nullsSize1).  slotSize is related to hash table (nullsSize2).  So it is more natural.

**Review by resetius — APPROVED** [review]

**Pull request merged**